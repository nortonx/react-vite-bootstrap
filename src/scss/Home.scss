.home {
  height: 72vh;
}