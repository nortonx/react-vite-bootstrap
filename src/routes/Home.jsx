import "../scss/Home.scss";
const Home = () => {
  return(
    <div className="container home" data-testid="home">
      <h1>This is the Home content</h1>
    </div>
  )
}

export default Home;