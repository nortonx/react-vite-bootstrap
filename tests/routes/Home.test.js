import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";

import Home from "../../src/routes/Home";

describe("Home route", () => {
  it("Should render the component", async () => {
    render(<Home />);
    const element = await screen.findByTestId("home");
    expect(element).toBeInTheDocument();
  });

  it("Should render and match snapshot", () => {
    const home = renderer.create(<Home />);
    expect(home).toMatchSnapshot();
  });
})