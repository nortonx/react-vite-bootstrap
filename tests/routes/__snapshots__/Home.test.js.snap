// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home route Should render and match snapshot 1`] = `
<div
  className="container home"
  data-testid="home"
>
  <h1>
    This is the Home content
  </h1>
</div>
`;
